line-colors.csv

```diff
@@ -276,17 +276,24 @@ vrn-sbahn-rn,S33,db-regio-ag-mitte,4-801539-33,#833aa1,#ffffff,,pill
 vrn-sbahn-rn,S39,db-regio-ag-mitte,4-801539-39,#ffffff,#bf5810,#bf5810,pill
 vrn-sbahn-rn,S44,db-regio-ag-mitte,4-801539-44,#ffffff,#00a650,#00a650,pill
 vrn-sbahn-rn,S51,db-regio-ag-mitte,4-801518-51,#f68a25,#ffffff,,pill
-vrr-rheinbahn-stadtbahn,U70,,7-vrr070-70,#71b2d0,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U71,,7-vrr070-71,#59c6f2,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U72,,7-vrr070-72,#25b8c5,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U73,,7-vrr070-73,#4465ad,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U74,,7-vrr070-74,#2c2276,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U75,,7-vrr070-75,#008fc2,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U76,,7-vrr070-76,#0063af,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U77,,7-vrr070-77,#7197cf,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U78,,7-vrr070-78,#009adf,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U79,,7-vrr070-79,#009a93,#ffffff,,rectangle
-vrr-rheinbahn-stadtbahn,U83,,7-vrr070-83,#1d3a8f,#ffffff,,rectangle
+vrr-rheinbahn,U70,,7-vrr070-70,#71b2d0,#ffffff,,rectangle
+vrr-rheinbahn,U71,,7-vrr070-71,#59c6f2,#ffffff,,rectangle
+vrr-rheinbahn,U72,,7-vrr070-72,#25b8c5,#ffffff,,rectangle
+vrr-rheinbahn,U73,,7-vrr070-73,#4465ad,#ffffff,,rectangle
+vrr-rheinbahn,U74,,7-vrr070-74,#2c2276,#ffffff,,rectangle
+vrr-rheinbahn,U75,,7-vrr070-75,#008fc2,#ffffff,,rectangle
+vrr-rheinbahn,U76,,7-vrr070-76,#0063af,#ffffff,,rectangle
+vrr-rheinbahn,U77,,7-vrr070-77,#7197cf,#ffffff,,rectangle
+vrr-rheinbahn,U78,,7-vrr070-78,#009adf,#ffffff,,rectangle
+vrr-rheinbahn,U79,,7-vrr070-79,#009a93,#ffffff,,rectangle
+vrr-rheinbahn,U83," ",7-vrr070-83,#1d3a8f,#ffffff,,rectangle
```

Comment:
```suggestion
vrr-rheinbahn,U83," ",7-vrr070-83,#1d3a8f,#ffffff,,rectangle
```

what is that " "? Why is that? :o

Reply:
interestingly enough it doesnt show that on my end, but ill try